components/tls/tls-mpc/src/follower.rs

```diff
@@ -216,6 +215,24 @@ impl MpcTlsFollower {
         Ok(())
     }
 
+    fn accepting_messages(&self) -> Result<(), MpcTlsError> {
+        if self.close_notify {
+            return Err(MpcTlsError::new(
+                Kind::PeerMisbehaved,
+                "attempted to commit a message after receiving CloseNotify",
```

Comment:
The problem with "commit" is that the Leader themselves don't know if they are "committing" to an authentic ciphertext until the MAC is verified.
Usually someone commits to a value that they know is authentic.

From the TLS-MPC client's point of view, it can be simply called "Receiving the TLS message". The fact that the leader receives it first and forwards it to the follower does not add any commitment properties to it, imo.
Both parties still need to authenticate the received message.